api/locale/de/LC_MESSAGES/django.po

```diff
@@ -706,155 +738,189 @@ msgid ""
 "This is an automated email, please do not reply. For any question, feel free "
 "to contact us at"
 msgstr ""
-"Dies ist eine automatisierte e-Mail, bitte antworten Sie nicht direkt auf diese e-Mail."
-" Bei Fragen kontaktieren Sie uns bitte unter:"
+"Dies ist eine automatisierte e-Mail, bitte antworten Sie nicht direkt auf "
+"diese e-Mail. Bei Fragen kontaktieren Sie uns bitte unter:"
 
 #: api/templates/email_download_ready.html:4
-#, python-format
-msgid " L'extraction pour la commande n°%(order_id)s a été réalisée. "
-msgstr " Die Daten für den Autrag n°%(order_id)s wurden bereit gestellt. "
+#, fuzzy, python-format
+#| msgid "The extraction for order n%(order_id)s has been completed. "
+msgid "The extraction for order n°%(order_id)s has been completed. "
+msgstr "Die Daten für den Autrag n°%(order_id)s wurden bereit gestellt. "
 
-#: api/templates/email_download_ready.html:6
+#: api/templates/email_download_ready.html:5
 msgid ""
-" Connectez-vous sur le géoshop et téléchargez les produits commandés dans la "
-"section \"Mes commandes\"\n"
-"        ou en cliquant directement sur ce lien: "
+"Log in to the Geoshop and download the ordered products from the \"My "
+"Orders\" section\n"
+"or by clicking directly on this link: "
 msgstr ""
-" Öffnen Sie den GeoShop und laden Sie die bestellten Produkte im "
-"Abschnitt \"Meine Bestellungen\"\n"
-"          oder durch Klick auf folgende Adresse: "
+"Öffnen Sie den GeoShop und laden Sie die bestellten Produkte im Abschnitt "
+"\"Meine Bestellungen\"\n"
+" oder durch Klick auf folgende Adresse: "
 
-#: api/templates/email_download_ready.html:13
-msgid ""
-"Veuillez prendre note que les liens de téléchargement ne sont disponibles "
-"que pendant 1 mois."
+#: api/templates/email_download_ready.html:10
+msgid "Please note that the download links are only available for 1 month."
 msgstr ""
-"Bitte beachten Sie das die bestellten Produkte nur während "
-"einem Monat herunter geladen werden können."
+"Bitte beachten Sie das die bestellten Produkte nur während einem Monat "
+"herunter geladen werden können."
 
 #: api/templates/email_password_reset.html:4
 #, python-format
 msgid ""
 "You're receiving this email because you requested a password reset for your "
 "user account at %(site_name)s."
 msgstr ""
-"Sie erhalten diese e-Mail, da Sie eine Passwort zurücksetzen Anfrage für Ihren "
-"Zugang bei %(site_name)s erstellt haben."
+"Sie erhalten diese e-Mail, da Sie eine Passwort zurücksetzen Anfrage für "
+"Ihren Zugang bei %(site_name)s erstellt haben."
 
 #: api/templates/email_password_reset.html:6
 msgid "Please go to the following page and choose a new password:"
 msgstr "Bitte gehen Sie zu folgender Seite and wählen ein neues Passwort:"
 
 #: api/templates/email_password_reset.html:10
+#, fuzzy
```

Comment:
also fuzzy (we don't need this function right now, but maybe it is easy to clean up?)

Reply:
Done, resolved it manually. Probably, the line was added by the tool "poedit"